### PR TITLE
Making Tutorial notebook easier to run on colab directly

### PR DESCRIPTION
**Pull request by jalammar**
To make the Tutorial run on Google Colab, I had to add the two commands to install dependencies:

```
!sudo apt-get install fluidsynth
!pip install pretty_midi mir_eval pyfluidsynth
```

In case it could make it easier for folks to run the Tutorial notebook, I've added them to the top (commented), and linked to the colab in the README. PR just in case it can make it easier for new users to quickly run for the first time.

[Images in the diff should be unchanged, not sure why/how the editor changed them]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 `pretty_midi` contains utility function/classes for handling MIDI data, so that it's in a format which is easy to modify and extract information from.
 
-Documentation is available [here](http://craffel.github.io/pretty-midi/).  You can also find a Jupyter notebook tutorial [here](https://nbviewer.org/github/craffel/pretty-midi/blob/main/Tutorial.ipynb).
+Documentation is available [here](http://craffel.github.io/pretty-midi/).  You can also find a Jupyter notebook tutorial [here](https://nbviewer.org/github/craffel/pretty-midi/blob/main/Tutorial.ipynb). [[Colab](https://colab.research.google.com/github/craffel/pretty-midi/blob/main/Tutorial.ipynb)]
```
Comment (craffel) [inline README.md:3]: Can you make this a little more grammatical, e.g. "You can also find a Jupyter notebook [here] (click [here] to load in Colab)." or something

File: Tutorial.ipynb [hunk 2]
```diff
@@ -181,7 +185,7 @@
...
...3evLjF6SJM1iohN9kn+X5PtJ7khyftfxLAcLveZJXp/kJ0lubn/e2EWcy02Sv07y4yS3dh3LcrHQa57khCT/b+i/hXcvdYzLTZL920Jpt7cF0d7adUzjYGJv3bfV7H4AvAy4B7gBOLOqbu80sB5bzGue5PXAUXMtbqTRSHIcsAn4m6o6pOt4loOFXvMkJwDnVNXJSx3bcpXkqcBTq+qmJI8DbgROW+55YZJn9EcDd1TVnVW1FbgceGXHMfWdr/mYqqprgX/pOo7lxNd8/FTVfVV1U/v7w8D3gF/tNqruTXKi/1XgR0Pb9+AbOmqLfc1/I8ktSa5oF0SSlqtj2hofX0nynK6DWU6SrAeeB/zvbiPp3iQneo2nq4H1VXUY8HXgkx3HI3XlJpplSp8LfAj4fMfxLBvtMuqfA942VPJ82ZrkRP9PwPBs8Wltn0Znwde8qh6sqi3t5mXAkUsUmzRWquqhqtrU/v5lYHVb2EsjlGQ1TZL/dFX9z67jGQeTnOhvAP5VkmckWUOz1O5VHcfUdwu+5u2HYQZOpXlGJi07SZ7SFvciydE0/799sNuo+q19vT8GfK+q3t91PONiYmu7VtVUkrOBvwVWAn9dVbd1HFavzfWaJ/kvwHer6irg95KcCkzRfFDp9Z0FvIwk+SxwAvCkJPcAf1xVH+s2qn6b7TUHVgNU1aXA6cBbkkwBPwNeW5P6NafJ8ULg3wMbk9zc9v1Be0dl2ZrYr9dJkqSFTfKte0mStAATvSRJPWailySpx0z0kiT1mIlekqQem9iv10na/ZL8CvCNdvMpwHbgJ+32T6vq1zoJTNIu8+t1kmaV5AJgU1W9r+tYJO06b91LWpQkm9r2hCR/n+QLSe5M8t+SnJXk+iQbkxzYjtsnyeeS3ND+vLDbf4G0PJnoJe2K5wJvBp5NsxLZs6rqaJr6Bv+5HfMXwAeq6vnAb7T7JC0xn9FL2hU3VNV9AEn+Afha278ROLH9/aXAv2mXewfYO8leg0IvkpaGiV7Srtgy9PuOoe0d/Pz/KyuAf1tVm5cyMEnTeete0qh8jZ/fxifJ4R3GIi1bJnpJo/J7wFFJbklyO80zfUlLzK/XSZLUY87oJUnqMRO9JEk9ZqKXJKnHTPSSJPWYiV6SpB4z0UuS1GMmekmSesxEL0lSj/1/d1wlwnWUcxUAAAAASUVORK5CYII=\n",
```
Comment (craffel) [inline Tutorial.ipynb:184]: Can you revert the changes to the data blobs? I think they're just noise.

**Review by craffel — CHANGES_REQUESTED** [review]
Thanks Jay! Some small changes requested.

File: Tutorial.ipynb (hunk 2 above)
Comment (jalammar) [reply Tutorial.ipynb]: Yeah, my bad. Should be good now.

File: README.md (hunk 1 above)
Comment (jalammar) [reply README.md]: Yeah! Done!

**Comment by jalammar** [comment]
Thank you! I was delighted to find the package and how easy it looks. I had always wanted some programmatic way to experiment with sounding out chords and chord progressions, and some freedom of moving MIDI around. Appreciate the effort you and all the contributors put into this!
